content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Membuat Pod Statis
+weight: 170
+content_type: task
+---
+
+<!-- overview -->
+
+Pod statis dikelola langsung oleh kubelet daemon pada suatu Node spesifik,
+tanpa {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+mengobservasi mereka.
+Tidak seperti Pod yang dikelola oleh control plane (contohnya,
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}});
+kubelet akan memantau setiap Pod statis (dan menjalankan ulang jika
+Pod mengalami kegagalan).
+
+Pod statis selalu terikat pada satu {{< glossary_tooltip term_id="kubelet" >}}
+pada Node spesifik.
+
+Kubelet secara otomatis akan mengulang untuk membuat sebuah
+{{< glossary_tooltip text="Pod mirror" term_id="mirror-pod" >}}
+pada Kubernetes API server untuk setiap Pod statis.
+Ini berarti Pod yang berjalan pada Node akan terlihat oleh API server,
+namun tidak dapat mengontrol darisana.
+
+{{< note >}}
+Jika kamu menjalankan Kubernetes terklaster dan menggunakan Pod statis
```

Comment:
"terklaster" doesn't seem to make sense, how about just klaster? unless there's special meaning
```suggestion
Jika kamu menjalankan klaster Kubernetes dan menggunakan Pod statis
```